### ISSUE TITLE
Create authentication

### ISSUE THREAD

**Issue by nguyenluke25** [G9-W2022-COMP231/OurRecipes#23]

**Comment by JameeKim** [comment]
Currently blocked by #24

**Comment by JameeKim** [comment]
Starting to work on this since #19 is blocked by other tasks

**Comment by JameeKim** [comment]
- Estimated: 1.5 hours
- Actual: 2 hours

Note:
When making the estimate, I only thought of creating a new user in the database and starting the user auth session. However, as I started working on this, I realized that our team did not make a separate task for handling the user input and sending appropriate response to the browser. It took more time than the estimate because I failed to fully identify the scope of this task.
